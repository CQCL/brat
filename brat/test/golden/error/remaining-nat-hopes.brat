show(n :: #) -> []
show(_) = []

read([]) -> n :: #
read([]) = 42

bad :: []
bad = let _ = read([]) in show(!)
