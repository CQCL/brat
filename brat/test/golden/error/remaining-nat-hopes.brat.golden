Error in test/golden/error/remaining-nat-hopes.brat@FC {start = Pos {line = 8, col = 31}, end = Pos {line = 8, col = 34}}:
bad = let _ = read([]) in show(!)
                              ^^^

  Expected to work out values for these holes:
    In checking_check_defs_1_bad_2__2 0


